lib/thread.h

```diff
@@ -240,6 +240,12 @@ extern void thread_call (struct thread *);
 extern unsigned long thread_timer_remain_second (struct thread *);
 extern struct timeval thread_timer_remain(struct thread*);
 extern int thread_should_yield (struct thread *);
+
+extern unsigned long timeval_elapsed (struct timeval a, struct timeval b);
```

Comment:
timeval_elapsed as some value as it returns a "long" while timersub requires to pass the result pointer as a variable which is not really practical when I want just to print a time difference.
Why not modifying timeval_elapsed to use timersub and move it to monotime.h ?